spec.md

```diff
@@ -868,6 +869,10 @@ The frequency and maximum duration of polling MAY vary by Platform client. If
 a Platform has a max polling duration and this limit is reached, the Platform
 MUST cease polling and the operation state MUST be considered `failed`.
 
+If a Service Offering declares a `maximum_polling_duration`
+in the [Catalog](#catalog-management)
+endpoint, that value SHOULD be used by the platform as its max polling duration.
```

Comment:
I think we should be more explicit here. Maybe something like:

```
The frequency and maximum duration of polling MAY vary by Platform. Additionally, Service Broker's can specify the maximum time a Platform SHOULD poll for via the `maximum_polling_duration` field returned by the [Catalog](#catalog-management) endpoint. If either the Platform or Service Broker's maximum polling duration is reached, the Platform SHOULD cease polling and the operation state MUST be considered failed.
```

Reply:
In that edit, you've changed the Platform's cease pooling (which predates my PR) from MUST to SHOULD. Was that intentional? It seems not strictly related to this change.

Reply:
That was intentional. It seemed weird to define a MUST for a Platform based on something the Platform decides!

Reply:
Gotcha. Yeah, definitely on board. That particular MUST was a bit weird.